docs/development/localization.md

```diff
@@ -78,6 +79,23 @@ About the interface:
 | positional-only            | только позиционный                                 |
 | keyword-only               | только именованный                                 |
 
+#### Формальные спецификаторы из `pyright`
+
+С определённого момента pyright начал поставлять формальные требования для некоторых сообщений:
+```
+        "noReturnContainsReturn": {
+            "message": "Function with declared return type \"NoReturn\" cannot include a return statement",
+            "comment": "{Locked='NoReturn','return'}"
+        },
+        "duplicateUnpack": {
+            "message": "Only one unpack operation allowed in list",
+            "comment": "{Locked='list'}"
+        },
+```
+
+Они оставлены в файлах локализации, чтобы было проще подтягивать изменения из pyright. Игнорируйте их.
+Часто они сами содержат смысловые ошибки (см. примеры выше). Для решения, переводить термин или нет, используйте глоссарий выше.
```

Comment:
i think this part would probably apply to all languages, so maybe it should be moved to the "general guidelines" section and written in english instead

i used google translate for this, but the part that says "To decide whether to translate a term or not, use the glossary above." could then be updated to say "refer to the language-specific guidelines below"